package/raphael.js

```diff
@@ -5711,6 +5717,11 @@
         next,
         prev,
         timestamp,
+        tempDiff,
+        tempDiff1,
+        tempDiff2,
+        tempDiff3,
+        change,
```

Comment:
Instead of creating a function, i have used an object to store the values and reduced the number of variables used.